ui/src/remote.ts

```diff
@@ -49,6 +54,10 @@ export const createStore = <T>(): Store<T> => {
   const internalStore = writable(initialState, set => {
     if (starter) {
       return starter(set);
+    } else {
+      return () => {
+        set({ status: Status.NotAsked });
+      };
```

Comment:
This behavior might be very confusing and has the potential to break a lot of things in the app since it is so widely used. I’d propose to make this behavior opt-in or leave it out entirely.

Reply:
Not resetting the store upon unsubscribe, has led to many issues with wrong/old state being rendered. It has been a bane and hard to debug. I'd argue the opposite is the case, not having this has led to unexpected behaviour. Given that the app is still functional and works as expected there is no apparent issue.

Reply:
Can you provide a concrete example where this fixes an issue? I’m ok with this change if you have confidence in it but I believe this might be more an issue with us using global stores. Some examples should help us figure this out.

Reply:
If you look at the parts in the source browsing that have been refactored, those relied heavily on a plethora of "reset" calls to enure that state would be reverted. These happen in many places away from the actual state handling and would rely on the ceremonial knowledge of the implementer to not be forgotten. If forgotten long winding debug session had to follow with patches putting a reset in the right place at the right time. Essentially this PR is a remedy of that larger issue by centralising state in screen specific stores and bundling data together where needed, while removing the need of consumers in components to know about the incantations required to avoid subtle breakages, double loading, weird flickering - all resulting from state being inconsistent.

Reply:
Thanks for the explanation.